buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -475,6 +475,11 @@ def get_pipeline_name(spec, version) {
 */
 def get_summary_table(identifier) {
     // fetch the downstream builds of the current build
+    try {
```

Comment:
What's the purpose of `safe_buildFile`? Is never used.
Could `buildFile` be `null` here? If null, than wrap it in a `try {} catch{}`

Reply:
If the build fails before the buildFile is loaded, the finally fails becasue buildFile is null and gives an error which misleads the user about why the build actually failed. I hit this issue when testing my changes but yes it is mostly unrelated to this PR. IMO this is a bit cleaner than wrapping the entire get_summary_table in try/catch.

Reply:
I will move this to a different PR so we don't hold up this change.